CONTRIBUTING.md

```diff
@@ -23,10 +23,12 @@ The project's Gemfile contains a few developer tools to help keep the project ti
 Declare model attributes in the following order:
 
  1. includes (e.g. `include UserPasswordConcern`)
- 2. callbacks
- 3. attribute overrides (e.g. `enum`s)
- 4. validations
- 5. relations
+ 2. declarations (e.g. `acts_as :superclass`, `stampable`)
+ 3. callbacks
+ 4. attribute overrides (e.g. `enum`s, workflows)
+ 5. validations
+ 6. associations
```

Comment:
..read the link in the following para =.=

Callbacks are inherited by subclasses. Declaring the relations after the callbacks allows the callbacks to be inherited.